reactioncommerce/reaction
# [Mini-EPIC] Orders List View

Resolves #2323 and #2325

#2323 Order list toggle
Users with permissions to view the list of orders should be able to toggle between list and detail view.
The list view is a "rolled up" view of each order but should still show the most relevant information:
- Order ID
- Customer Name
- Order Status
- Order Date
- Order Risk Assessment if elevated or high
- Transit Status if SHIPPED

**Testing**
- Clone this branch
- Have at least 3 orders made in your shop
- Visit the orders page on the dashboard
- Use the toggle buttons on the top to toggle between the list and detail view
- Make some more orders to test that pagination works correctly on the list view
- Test that you can complete an order successfully from the list view
- Test that you can select/deselect rows by clicking on the checkboxes

#2325 Select Orders

Users with permissions to view orders are able to select/deselect orders within the order list, whether one by one or all the orders listed in the current page.

**Testing**
- Clone this branch
- Have at least 3 orders made in your shop
- Visit the orders page on the dashboard
- Hover over an avatar while in the orders list. Notice on hover that the avatar rolls over to a check box
- Select the check box. Notice that a bar appears over the table that has the number of selected orders
- Select the check box in the bar that appears over the table. Notice that all the orders you have listed are now selected. Clicking on the checkbox again deselects all the orders.

N.B: The two buttons that appear on the bar (the `Capture` and `Bulk Actions` buttons) are not functional as at this point. Functionality for them will be handled in #2341
